sql/core/src/test/scala/org/apache/spark/sql/JoinHintSuite.scala

```diff
@@ -281,7 +281,7 @@ class JoinHintSuite extends PlanTest with SharedSQLContext {
           JoinHint(
             Some(HintInfo(strategy = Some(SHUFFLE_HASH))),
             Some(HintInfo(strategy = Some(SHUFFLE_HASH)))) :: Nil,
-        msgNoHintRelationFound("d", "merge(b, d)") ::
+        msgNoHintRelationFound("d", "merge('b, 'd)") ::
```

Comment:
it's weird to see `'b` representing a table name, `'b` usually represents a column.

Reply:
That's the default output from the hint parameters. I simply called `toString` for all the parameters.